cpp/src/fil/infer.cu

```diff
@@ -36,7 +36,7 @@
 #endif  // __CUDA_ARCH__
 #endif  // CUDA_PRAGMA_UNROLL
 
-#define INLINE_CONFIG __forceinline__
+#define INLINE_CONFIG __noinline__
```

Comment:
Is this change meant to be temporary, or is it required for float64 to work?

Reply:
I've reverted it.

Thanks for noticing!